Gordon360/Services/HousingService.cs

```diff
@@ -111,20 +111,20 @@ public AA_ApartmentHalls[] GetAllApartmentHalls()
         /// Calls a stored procedure that tries to get the id of an the application that a given user is 
         /// applicant on for a given session
         /// </summary>
-        /// <param name="userID"> The student username to look for </param>
+        /// <param name="username"> The student username to look for </param>
         /// <param name="sess_cde"> Session for which the application would be </param>
         /// <returns> 
         /// The id of the application or 
         /// null if the user is not on an application for that session 
         /// </returns>
-        public int? GetApplicationID(string userID, string sess_cde)
+        public int? GetApplicationID(string username, string sess_cde)
         {
             IEnumerable<ApartmentAppIDViewModel> idResult = null;
 
-            SqlParameter userParam = new SqlParameter("@STUDENT_ID", userID);
+            SqlParameter userParam = new SqlParameter("@STUDENT_USERNAME", username);
             SqlParameter sessionParam = new SqlParameter("@SESS_CDE", sess_cde);
 
-            idResult = RawSqlQuery<ApartmentAppIDViewModel>.query("GET_AA_APPID_BY_STU_ID_AND_SESS @SESS_CDE, @STUDENT_ID", sessionParam, userParam); //run stored procedure
+            idResult = RawSqlQuery<ApartmentAppIDViewModel>.query("GET_AA_APPID_BY_STU_ID_AND_SESS @SESS_CDE, @STUDENT_USERNAME", sessionParam, userParam); //run stored procedure
```

Comment:
Has this SP been renamed/should it be renamed?
```suggestion
            idResult = RawSqlQuery<ApartmentAppIDViewModel>.query("GET_AA_APPID_BY_USERNAME_AND_SESS @SESS_CDE, @STUDENT_USERNAME", sessionParam, userParam); //run stored procedure
```